package.json

```diff
@@ -14,6 +14,7 @@
     "storybook": "pnpm run --parallel --recursive storybook:run",
     "storybook:build": "npm run build && turbo run storybook:build",
     "storybook:ds:build": "turbo run storybook:build --filter=@webstudio-is/design-system",
+    "storybook:builder:build": "turbo run storybook:build --filter=@webstudio-is/builder",
```

Comment:
we need something better than a separate command per package, because you don't want this to become a list of 10-20 packages

Reply:
Fixed this with a script.